src/algokit/__main__.py

```diff
@@ -1,3 +1,10 @@
+from multiprocessing import freeze_support
+
 from algokit.cli import algokit
+from algokit.cli.common.utils import is_binary_mode
+
+# Required to support full feature parity when running in binary execution mode
+if is_binary_mode():
```

Comment:
It seems from the docs we can omit this `if` check and just always run `freeze_supprt`? https://docs.python.org/3/library/multiprocessing.html#multiprocessing.freeze_support